connectors/src/start_worker.ts

```diff
@@ -13,24 +16,48 @@ import logger from "./logger/logger";
 
 setupGlobalErrorHandler(logger);
 
-runConfluenceWorker().catch((err) =>
-  logger.error(errorFromAny(err), "Error running confluence worker")
-);
-runSlackWorker().catch((err) =>
-  logger.error(errorFromAny(err), "Error running slack worker")
-);
-runNotionWorker().catch((err) =>
-  logger.error(errorFromAny(err), "Error running notion worker")
-);
-runGithubWorker().catch((err) =>
-  logger.error(errorFromAny(err), "Error running github worker")
-);
-runGoogleWorker().catch((err) =>
-  logger.error(errorFromAny(err), "Error running google worker")
-);
-runIntercomWorker().catch((err) =>
-  logger.error(errorFromAny(err), "Error running intercom worker")
-);
-runWebCrawlerWorker().catch((err) =>
-  logger.error(errorFromAny(err), "Error running webcrawler worker")
-);
+const ALL_WORKERS: ConnectorProvider[] = [
+  "confluence",
+  "slack",
+  "notion",
+  "github",
+  "google_drive",
+  "intercom",
+  "webcrawler",
+];
+
+async function runWorkers(workers: ConnectorProvider[]) {
+  const workerFunctions: Record<ConnectorProvider, () => Promise<void>> = {
```

Comment:
nit: we could avoid having both "ALL_WORKERS and this `Record` by having just a single top-level record  and treat Object.keys(record) as `ALL_WORKERS`.

Reply:
Fair point, updated.